stubs/cmr/queries.pyi

```diff
@@ -48,8 +48,8 @@ class GranuleCollectionBaseQuery(Query):
     def online_only(self, online_only: bool = True) -> Self: ...
     def temporal(
         self,
-        date_from: Optional[Union[str, datetime]],
-        date_to: Optional[Union[str, datetime]],
+        date_from: Optional[Union[str, date, datetime]],
+        date_to: Optional[Union[str, date, datetime]],
```

Comment:
Do we get to delete our stubs when python-cmr v0.11.0 is released?

Reply:
Yes, which reminds me, I forgot to ask when that will happen.